moebot_bot/bot/commands/commands.go

```diff
@@ -0,0 +1,31 @@
+package commands
+
+import (
+	"github.com/bwmarrin/discordgo"
+)
+
+type CommPackage struct {
+	session *discordgo.Session
+	message *discordgo.Message
+	guild   *discordgo.Guild
+	member  *discordgo.Member
+	channel *discordgo.Channel
+	params  []string
+}
+
+type Command interface {
+	Execute(pack *CommPackage)
+	Setup(session *discordgo.Session)
+	EventHandlers() []interface{}
+}
+
+func NewCommPackage(session *discordgo.Session, message *discordgo.Message, guild *discordgo.Guild, member *discordgo.Member, channel *discordgo.Channel, params []string) CommPackage {
```

Comment:
I don't really see the need for this function when we could just pass in the object literal to begin with, unless we were doing something extra (or planned to do something extra later)

Reply:
I'll fix this up in develop